Update prepare-ad-and-domains.md (#1426)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/plan-and-deploy/prepare-ad-and-domains.md

E:\Setup.exe /IAcceptExchangeServerLicenseTerms /PrepareSchema
```

> [!NOTE]

> > When you run the above command, a prerequisite check is performed and will tell you which computer requirements are missing.

Reviewer: ```suggestion
> When you run this command, a prerequisite check will tell you which requirements are still needed.
```

> > When you run the above command, a prerequisite check is performed and will tell you which computer requirements are missing.

Reviewer: ```suggestion
> When you run this command, a prerequisite check is performed that will tell you which requirements are missing.
```
